emcc.py

```diff
@@ -2043,7 +2049,7 @@ def get_final():
           # For the wasm backend, we don't have any memory info in JS. All we need to do
           # is set the memory initializer url.
           src = open(final).read()
-          src = src.replace('var memoryInitializer = null;', 'var memoryInitializer = "%s";' % memfile)
+          src = src.replace('var memoryInitializer = null;', 'var memoryInitializer = "%s";' % os.path.basename(memfile))
```

Comment:
How did this work before?

Reply:
It didn't - we just don't have full testing for pthreads yet, and that's the only code path that would notice.